src/Adapter/ContextStateManager.php

```diff
@@ -46,6 +47,7 @@
  */
 final class ContextStateManager
 {
+    //@todo: add private constant for each field?
```

Comment:
ping @jolelievre do you think its worth it?
I would say its better practice than leaving magic strings all over the place?

Reply:
Yep, good idea